python/docs/pyspark.ml.rst

```diff
@@ -33,6 +33,14 @@ pyspark.ml.classification module
     :undoc-members:
     :inherited-members:
 
+pyspark.ml.clustering module
+--------------------------------
```

Comment:
This dashed line should be exactly the same length as the above line, to avoid style complaints when compiling docs.